pkg/audit/manager.go

```diff
@@ -343,10 +367,10 @@ func (am *Manager) getAllConstraintKinds() ([]schema.GroupVersionKind, error) {
 	return ret, nil
 }
 
-func (am *Manager) getUpdateListsFromAuditResponses(res []*constraintTypes.Result) (map[string][]auditResult, map[string]int64, map[util.EnforcementAction]int64, error) {
-	updateLists := make(map[string][]auditResult)
-	totalViolationsPerConstraint := make(map[string]int64)
-	totalViolationsPerEnforcementAction := make(map[util.EnforcementAction]int64)
+func (am *Manager) addAuditResponsesToUpdateLists(
+	updateLists map[string][]auditResult,
+	res []*constraintTypes.Result,
+	totalViolationsPerConstraint map[string]int64, totalViolationsPerEnforcementAction map[util.EnforcementAction]int64) error {
 	// resetting total violations per enforcement action
 	for _, action := range util.KnownEnforcementActions {
 		totalViolationsPerEnforcementAction[action] = 0
```

Comment:
Since `addAuditResponsesToUpdateLists` is called on every violating object now, we don't want to reset  `totalViolationsPerEnforcementAction` every time right?

Reply:
good catch, updated